Docker/CI.Ubuntu22.dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM ubuntu:22.04
+
+# basic dependencies -
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt-get update -y && \
+  apt-get install -y 
+
+RUN apt-get install -y curl \
+  pkg-config \
+  findutils \
+  wget
+
+# build dependencies
+RUN apt-get install -y && \
+    apt-get remove -y llvm-12 clang-format-11 clang-tidy-11 && \
+    apt-get install -y curl pkg-config findutils  wget && \
+    apt install -y clang-13 cmake clang-format-13 clang-tidy-13 && \
+    apt-get install -y llvm-13 lldb-13 llvm-13-dev libllvm11 llvm-13-runtime opt && \
+    apt-get install -y nodejs npm && \
+    apt install -y python3 python3-pip && \
+    npm install -g prettier@2.2.1 && \
+    export CC=clang-13 && \
+    export CXX=clang++ 
+
+# Python
+RUN apt install -y python3 python3-pip && \
+    update-alternatives --install /usr/bin/python python /usr/bin/python3 0
+
+ADD ./requirements.txt /src/
+ADD ./manage /src/
```

Comment:
(nit, I don't know Docker) I assume that `ADD` _copies_ the files. What will be lost if the files are _moved_ instead?

Reply:
The command `ADD` copies a file from the host machine to the docker image ("VM" image). If this would be replaced with a move instruction, the data on the host machine would vanish which means that you'd not be able to run the build process twice in a row. I am not aware that there is an alternative to `ADD` but even if there was, I would hesitate using it because of the previous explanation.